### DISCUSSION TITLE
Exclude subpath from scope

### DISCUSSION THREAD

**Discussion by mattvb91** [shadowwalker/next-pwa#279] [Q&A]
Hey everyone,

Just wondering if & how it would be possible to exclude subpaths from the PWA scope. For example if my scope is `/` which includes `/app` in my app, is it possible to completely exclude a subpath like `/app/admin` from being built into the PWA?

Cheers

**Comment by rimzzlabs** [comment]
> Hey everyone,
> 
> Just wondering if & how it would be possible to exclude subpaths from the PWA scope. For example if my scope is `/` which includes `/app` in my app, is it possible to completely exclude a subpath like `/app/admin` from being built into the PWA?
> 
> Cheers

I'm working on it too, the scope property could be possibly accept an array of paths? Idk.

**Reply by mattvb91** [reply]
Heres how I ended up doing it but this is definitely a hack: 

```
 buildExcludes: [
        /middleware-manifest\.json$/,
        /_middleware\.js$/,
        /chunks\/pages\/app\/admin.*/, // <--- remove admin section from pwa
      ],
```
